setup.py

```diff
@@ -33,7 +33,7 @@
     packages=find_packages(exclude=['tests', 'docs']),
     include_package_data=True,
     zip_safe=False,
-    install_requires=['django-model-utils'],
+    install_requires=['django-model-utils', 'djangorestframework'],
```

Comment:
not all the packages using `openwisp-utils` have APIs and adding this here would make them download DRF even when not needed.

I suggest the following solution:

- remove this requirement here
- add djangorestframework to `requirements-test.txt`
- add a try/except `ImportError` block when you import `serializers` in `openwisp_utils/api/serializers.py`, see my other comment